docs/telegram-usage.md

```diff
@@ -171,8 +171,8 @@ official commands. You can ask at any moment for help with `/help`.
 | `/locks` | Show currently locked pairs.
 | `/unlock <pair or lock_id>` | Remove the lock for this pair (or for this lock id).
 | `/profit [<n>]` | Display a summary of your profit/loss from close trades and some stats about your performance, over the last n days (all trades by default)
-| `/forceexit <trade_id>` | Instantly exits the given trade  (Ignoring `minimum_roi`).
-| `/forceexit all` | Instantly exits all open trades (Ignoring `minimum_roi`).
+| `/force_exit <trade_id>` | Instantly exits the given trade  (Ignoring `minimum_roi`).
+| `/force_exit all` | Instantly exits all open trades (Ignoring `minimum_roi`).
```

Comment:
The alias is fine (should be added here, too) - but i'd still keep it at `/forceexit`, not `/force_exit`.

That aligns it to the majority of other commands (like `/forcelong` and `/forceshort`) - and using underscores on a phone is simply bad UX.

Reply:
Thinking about the alias a bit more - `/fe` could be both `forceentry` or `forceexit` - so it might actually be missleading ...

Reply:
> Thinking about the alias a bit more - `/fe` could be both `forceentry` or `forceexit` - so it might actually be missleading ...

What about /fx?

Reply:
> Thinking about the alias a bit more - `/fe` could be both `forceentry` or `forceexit` - so it might actually be missleading ...

How about /fx?

Reply:
yeah - that'll work :+1:

Reply:
> using underscores on a phone is simply bad UX.

What about for /reload_config?